charts/stable/projectsend/values.yaml

```diff
@@ -43,11 +43,11 @@ workload:
             - -c
           args:
             - |-
-              export configFile="/config/includes/sys.config.php";
+              export configFile="/config/projectsend/includes/sys.config.php";
```

Comment:
Is this correct? 
The file linked from /app/www/public/includes/sys.config.php is /config/projectsend/sys.config.php